addon/components/upf-rating.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  classNames: ['upf-rating'],
+
+  actions: {
+    saveRatingValue(rating) {
+      this.influencer.set('rating', rating);
```

Comment:
OSS is not supposed to be aware of the `influencer`, you should only pass it an object and that's it

Reply:
and where is that action triggered ?

Reply:
Oh yes, that's no longer used, forgot to remove it
Now I pass the callback function as a parameter